packages/dso-toolkit/src/styles/mixins/_mixins.scss

```diff
@@ -15,6 +15,7 @@
   width: 1px;
 }
 
+// Also moved to @dso-toolkit/styling/mixins/anchor
```

Comment:
Wat betekent dit?

Reply:
Ik had deze mixin nodig voor de styling in de breadcrumbs, maar ik durfde hem nog niet volledig te verplaatsen. Dat leek me meer bij een 'anchor' issue te horen.

Reply:
Daar was ik al bang voor, dat blokkeert dit issue dan dus?

Reply:
Nee hoor, ik kan ermee leven als je 'm in dit issue op de juiste manier verhuisd. Maar het is één van de twee:
* Eerst andere issue oppakken waarin deze mixin wordt verhuisd
* In deze issue mixin op de juiste manier verhuizen

Maar nu zit je ertussen en zijn er twee mixins actief.

Reply:
Impact analyse gaf aan dat er eigenlijk geen problemen waren. Ik heb de boel verhuisd.

Reply:
Mooi!